examples/kafka-consumer-service/kafka-consumer-service.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on creating a Kafka consumer to receive messages asynchronously using Ballerina.
```

Comment:
Suggested edit: BBE on creating a Kafka consumer to receive messages in an asynchronous manner using Ballerina.

(asynchronous has better SEO value than asynchronously)